synctl

```diff
@@ -307,12 +334,21 @@ def main():
             for cache_name, factor in iteritems(worker.cache_factors):
                 os.environ["SYNAPSE_CACHE_FACTOR_" + cache_name.upper()] = str(factor)
 
-            start_worker(worker.app, configfile, worker.configfile)
+                errors += int(start_worker(worker.app, configfile, worker.configfile))
 
             # Reset env back to the original
             os.environ.clear()
             os.environ.update(env)
 
+        process_count = len(workers) + int(start_stop_synapse)
+
+        # If every process failed to start we exit with exit code 1. If a subset
```

Comment:
I'm not really sure this is necessary. I'd just make it exit(1) if any of the processes failed.

Reply:
If that's the decision then I'll remove the int casts.